src/Resources/ShipmentSurcharge.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\ApiSdk\Resources;
+
+use MyParcelCom\ApiSdk\Resources\Interfaces\ResourceInterface;
+use MyParcelCom\ApiSdk\Resources\Interfaces\ShipmentSurchargeInterface;
+use MyParcelCom\ApiSdk\Resources\Traits\JsonSerializable;
+use MyParcelCom\ApiSdk\Resources\Traits\Resource;
+
+class ShipmentSurcharge implements ShipmentSurchargeInterface
+{
+    use JsonSerializable;
+    use Resource;
+
+    const AMOUNT = 'amount';
+    const CURRENCY = 'currency';
+
+    private ?string $id = null;
+
+    private string $type = ResourceInterface::TYPE_SHIPMENT_SURCHARGE;
+
+    private ?string $name = null;
+
+    private ?string $description = null;
+
+    private array $fee = [
+        self::AMOUNT   => null,
+        self::CURRENCY => null,
+    ];
+
+    public function setName(?string $name): self
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    public function getName(): ?string
+    {
+        return $this->name;
+    }
+
+    public function setDescription(?string $description): self
+    {
+        $this->description = $description;
+
+        return $this;
+    }
+
+    public function getDescription(): ?string
+    {
+        return $this->description;
+    }
+
+    public function setFeeAmount(int $amount): self
+    {
+        $this->fee[self::AMOUNT] = $amount;
+
+        return $this;
+    }
+
+    public function getFeeAmount(): int
```

Comment:
You have a good point. At the moment not a problem, surcharges are read-only, retrieved via a Shipment.

However, there are [CRUD endpoints](https://api-specification.myparcel.com/#tag/ShipmentSurcharges) to manage surcharges, which I think we should implement as well.

I can draft this PR and add a commit to introduce create/update/delete surcharges. Or make a follow-up PR?

Reply:
Will adding this code increase the PR size significantly, or is most of the code already there? If the PR doubles in size I would prefer a follow up

Reply:
Instead of adding the management endpoints, I decided to wrap up this read-only-surcharges PR.

I have added one more commit that adds `includes` to the Shipment model. This way, no additional requests are needed when SDK users call `$shipment->getShipmentSurcharges()` (and other relationship functions).